internal/emscripten/emscripten.go

```diff
@@ -86,10 +89,8 @@ func (v *InvokeFunc) Call(ctx context.Context, mod api.Module, stack []uint64) {
 		panic(err)
 	}
 
-	ret, err := m.Engine.NewFunction(idx).Call(ctx, params...)
+	err = m.Engine.NewFunction(idx).CallWithStack(ctx, stack)
```

Comment:
Found the non-test/non-example usage. Also neatly validates the design: we're using `CallWithStack` to implement a `GoModuleFunction`.

This should have a real world impact on @jerbob92's emscripten uses.

Reply:
hehe except I'm not sure he uses this part anymore. can you also hit this one? https://github.com/tetratelabs/wazero/blob/main/internal/emscripten/emscripten.go#L89

Reply:
disregard I hadn't coffee yet ;)